### PR TITLE
test html plugin url radio

### PR DESCRIPTION
**Pull request by ShamaySapir**

### CODE REVIEW

File: e2e/cypress/support/commands.js [hunk 1]
```diff
@@ -516,16 +516,27 @@ Cypress.Commands.add('clickOnPluginMenuButton', dataHook =>
   cy.get(`[data-hook*=addPluginMenu] [data-hook*=${dataHook}]`).click({ force: true })
 );
 
-Cypress.Commands.add('addHtml', () => {
-  cy.clickOnStaticButton(HTML_PLUGIN.STATIC_TOOLBAR_BUTTON);
-  cy.get(`[data-hook*=${HTML_PLUGIN.INPUT}]`)
+Cypress.Commands.add('addHtmlPlugin', (input, data) => {
```
Comment (shaulgo) [inline e2e/cypress/support/commands.js:519]: this should just be a function. It's only used in this file.

File: e2e/cypress/support/commands.js [hunk 2]
```diff
@@ -516,16 +516,27 @@ Cypress.Commands.add('clickOnPluginMenuButton', dataHook =>
...
+  cy.get(`[data-hook*=${input}]`)
     .click()
     .clear();
```
Comment (shaulgo) [inline e2e/cypress/support/commands.js:522]: I'm not sure exactly what is happening here. The data-hook for the text input should be the same for both html and url.

**Review by shaulgo — CHANGES_REQUESTED** [review]

File: e2e/cypress/dataHooks.js [hunk 3]
```diff
@@ -99,6 +99,8 @@ export const HTML_PLUGIN = {
   STATIC_TOOLBAR_BUTTON: 'HTMLCodePlugin_InsertButton',
   INPUT: 'htmlEditPanel_htmlInput',
   UPDATE: 'htmlEditPanel_Update',
+  URL: 'htmlEditPanel_radioUrl',
```
[the remark this answers is not in the document]
Comment (ShamaySapir) [reply e2e/cypress/dataHooks.js:102]: fixed

File: e2e/cypress/support/commands.js (hunk 1 above)
Comment (ShamaySapir) [reply e2e/cypress/support/commands.js:519]: fixed

File: e2e/cypress/support/commands.js (hunk 2 above)
Comment (ShamaySapir) [reply e2e/cypress/support/commands.js:522]: fixed

File: e2e/cypress/support/commands.js [hunk 4]
```diff
@@ -516,16 +516,26 @@ Cypress.Commands.add('clickOnPluginMenuButton', dataHook =>
   cy.get(`[data-hook*=addPluginMenu] [data-hook*=${dataHook}]`).click({ force: true })
 );
 
-Cypress.Commands.add('addHtml', () => {
-  cy.clickOnStaticButton(HTML_PLUGIN.STATIC_TOOLBAR_BUTTON);
+function addHtmlPlugin(data) {
```
Comment (shaulgo) [inline e2e/cypress/support/commands.js:519]:
```suggestion
function addHtmlPlugin(data, isUrl) {
	 cy.clickOnStaticButton(HTML_PLUGIN.STATIC_TOOLBAR_BUTTON);
	 if (isUrl) {
	   cy.get(`[data-hook*=${HTML_PLUGIN.RADIO_URL}]`).click();
	 }
	 ...
```

Comment (ShamaySapir) [reply e2e/cypress/support/commands.js]: fixed

**Review by shaulgo — APPROVED** [review]

**Pull request merged**